R/Mexico.R

```diff
@@ -147,14 +149,16 @@ Mexico <- R6::R6Class("Mexico",
           date = dmy(.data$date)
         ) %>%
         select(-.data$nombre) %>%
-        full_join(self$data$codes_lookup, by = "inegi_state") %>%
+        full_join(self$region_codes[["level_1_region"]],
```

Comment:
can we join using `region_level_1` or do we need region codes to contain the localised version of the code?